gradle-scalatest.plugin

```diff
@@ -0,0 +1,21 @@
+buildscript {
+	repositories {
+		maven {
+			url "https://plugins.gradle.org/m2/"
+		}
+	}
+	dependencies {
+		classpath "com.github.maiflai:gradle-scalatest:0.8"
+	}
+}
+
+if (!project.plugins.findPlugin(com.github.maiflai.ScalaTestPlugin)) {
```

Comment:
First saw this in https://raw.githubusercontent.com/akhikhl/gradle-onejar/master/pluginScripts/gradle-onejar.plugin, looked like good practice. Is there a down-side to this being here?